Integrations/Cybereason/Cybereason.py

```diff
@@ -0,0 +1,1478 @@
+    client_key_file = os.path.abspath('client.pem')
+
+    session.cert = (client_cert_file, client_key_file)
+
+    # Time to check if we are logged on
+    response = session.get(url=SERVER)
+    if response.status_code != 200 and response.status_code != 302:
+        return_error("Failed to connect to server")
+
+    # First time we may get a redirect, but second time should be 200
+    response = session.get(url=SERVER)
+    if response.status_code != 200:
+        return_error("Failed to login with certificate. Expected response 200. Got: " + str(response.status_code))
+
+
+def logout():
+    cmd_url = '/logout'
+    http_request('GET', cmd_url)
+
+
+''' EXECUTION CODE '''
+
+LOG('command is %s' % (demisto.command(), ))
+
+session = requests.session()
+
+
+if CERTIFICATE:
+    client_certificate()
+    AUTH = 'CERT'
+if USERNAME and PASSWORD:
+    login()
+    AUTH = 'BASIC'
+else:
+    return_error('No credentials were provided')
+
+try:
+    if demisto.command() == 'test-module':
+        # Tests connectivity and credentails on login
+        query_user([])
+        demisto.results('ok')
+
+    elif demisto.command() == 'fetch-incidents':
+        fetch_incidents()
+
+    elif demisto.command() == 'cybereason-is-probe-connected':
+        is_probe_connected_command()
+
+    elif demisto.command() == 'cybereason-query-processes':
+        query_processes_command()
+
+    elif demisto.command() == 'cybereason-query-malops':
+        query_malops_command()
+
+    elif demisto.command() == 'cybereason-query-connections':
+        query_connections_command()
+
+    elif demisto.command() == 'cybereason-isolate-machine':
+        isolate_machine_command()
+
+    elif demisto.command() == 'cybereason-unisolate-machine':
+        unisolate_machine_command()
+
+    elif demisto.command() == 'cybereason-malop-processes':
+        malop_processes_command()
+
+    elif demisto.command() == 'cybereason-add-comment':
+        add_comment_command()
+
+    elif demisto.command() == 'cybereason-update-malop-status':
+        update_malop_status_command()
+
+    elif demisto.command() == 'cybereason-prevent-file':
+        prevent_file_command()
+
+    elif demisto.command() == 'cybereason-unprevent-file':
+        unprevent_file_command()
+
+    elif demisto.command() == 'cybereason-kill-process':  # To be added as a command in the future
+        kill_process_command()
+
+    elif demisto.command() == 'cybereason-quarantine-file':  # To be added as a command in the future
+        quarantine_file_command()
+
+    elif demisto.command() == 'cybereason-delete-registry-key':  # To be added as a command in the future
+        delete_registry_key_command()
+
+    elif demisto.command() == 'cybereason-query-file':
+        query_file_command()
+
+    elif demisto.command() == 'cybereason-query-domain':
+        query_domain_command()
+
+    elif demisto.command() == 'cybereason-query-user':
+        query_user_command()
+
+except Exception, e:
```

Comment:
error handling should be `return_error` and not `raise`.
```suggestion
except Exception as e:
```